packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts

```diff
@@ -116,6 +118,28 @@ export const V1_parameterValueModelSchema = createModelSchema(
   },
 );
 
+const V1_servicePostValidationAssertionModelSchema = createModelSchema(
+  V1_PostValidationAssertion,
+  {
+    id: primitive(),
+    assertion: usingModelSchema(V1_rawLambdaModelSchema),
```

Comment:
Please arrange them in alphabetical order

Reply:
> Can we add some tests?

one roundtrip test was added. thanks!